src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,21 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $logLevel
+     *
+     * @return $this
+     */
+    public function setLogLevel($logLevel)
```

Comment:
If we leave this out of the collection builder, then we can't call this method on taskExec. The `taskExec()` method creates a task via the TaskAccessor, which uses the CollectionBuilder.